### ISSUE TITLE
Weird Issue with text and textures

### ISSUE THREAD

**Issue by BigManGeorge2003** [leoetlino/project-restoration#183]
Text is blocked out by rectangles. Rectangles typically appear to be the same colour as the text is supposed to be. Textures also don't load in properly, some of them are low res and some of them have black backgrounds. Theres also some weird effect with the lighting. Not sure what thats all about.

Also, the game completely freezes when the first cutscene transitions into gameplay, it didn't do this the first time round so i'm not sure what that's all about, but the weird textures showed up in the gameplay section in the lost woods before you enter termina, so I couldn't get a screenshot of those.

No idea how to reproduce, this happened on my first launch of the mod. I followed all installation steps for the high res texture mod and the high res HUD mod.

**Screenshots**
![image](https://user-images.githubusercontent.com/85046134/120090690-511bcc80-c0fc-11eb-8c30-0b04d3708ba6.png)
![image](https://user-images.githubusercontent.com/85046134/120090750-b53e9080-c0fc-11eb-94ca-ff33a95bfeaf.png)
![image](https://user-images.githubusercontent.com/85046134/120090756-cbe4e780-c0fc-11eb-920c-0149808f4a7c.png)
![image](https://user-images.githubusercontent.com/85046134/120090838-8ffe5200-c0fd-11eb-9a23-6ff2b8a4761f.png)
(the last one is where game freezes)

**Additional context**
I'm running the most recent version of citra. the game ROM is one I downloaded, it seems to be MM3d 1.0 since thats what version of the mod files worked. 

I also installed textures from the MM3DHD project as reccomended on the website. Deleted them in case they were the problem and that didn't change anything.

PC specs, in case thats important: Windows 10 32bit home edition, ryzen 5 3600x cpu, asus geforce GTX 1660 super GPU, 32gbs of RAM.

**Comment by DeathWrench** [comment]
I'm assuming your rom is corrupted

**Comment by leoetlino** [comment]
Yeah, this looks like a Citra or graphics driver issue.

**Comment by BigManGeorge2003** [comment]
Thanks for the help guys. not sure what was happening with the game freezing but thats fixed now. Turns out I hadn't uninstalled the HD textures mod like I thought, thats where all the issues are happening. I've posted on their github about the issue so it should be all fixed up soon.

BTW, I fully updated citra and my graphics drivers. made no difference. Installed a new rom as well. The MM3D restoration mod works perfectly on it's own. great job with it
